docs/cn/build_and_run.md

```diff
@@ -30,16 +30,26 @@ docker build -t opencurvedocker/curve-base:build-debian9
 ### 在docker镜像中编译
 
 ```bash
-docker run -it opencurvedocker/curve-base:build-debian9 /bin/bash
-cd <workspace>
 git clone https://github.com/opencurve/curve.git 或者 git clone https://gitee.com/mirrors/curve.git
+cd curve
+# 如果你想在容器内完成编译+制作+上传镜像的操作，可以添加以下参数
+# -v /var/run/docker.sock:/var/run/docker.sock -v /root/.docker:/root/.docker
+docker run -v $(pwd):/curve -w /curve -v /root/.cache/bazel:/root/.cache/bazel  --privileged -it opencurvedocker/curve-base:build-debian9 bash
```

Comment:
```suggestion
docker run -it --rm -v $(pwd):/curve -w /curve -v ${HOME} /.cache/bazel:/root/.cache/bazel -u $(id -u):$(id -g) opencurvedocker/curve-base:build-debian9 /bin/bash
```

What about bind-mounting build cache to the home dir of current user instead of root? So that we can possibly share cache.

User is kept the same as the user on host.

`--rm` deletes the container once we are done, instead of creating a new one everytime. There is no need to preserve runtime-data since everything is bind-mounted.

Reply:
Is `--rm` necessary? If you are a docker proficient user, you can continue to use this container to complete some work through exec or attach.

Reply:
Yes, you are right. Without `rm`, the user can start the container and reuse it.

The reason to use `rm` is that it seems this container acts as an ephemeral container, or like Jobs, and doesn't store persistent data. So I think creating a new one is fine in this case. Since `bash` is not a long-running command, so the container will exit once `bash` exits, and we cannot attach to it before it is started manually again.

Reply:
`bind-mounting build cache to the home dir of current user` is good idea.

Reply:
> Yes, you are right. Without `rm`, the user can start the container and reuse it. That's fine if they know how to do it.
> 
> The reason to use `rm` is that it seems this container acts as an ephemeral container, or like Jobs, and doesn't store persistent data. It is mostly used as a one-time job. So I think creating a new one is fine in this case. Since `bash` is not a long-running command, so the container will exit once `bash` exits, and we cannot attach to it before it is started manually again.

It can also be added to make a prompt. If the user wants to do some work such as testing, a container that is deleted after exiting is not friendly.

Reply:
That makes sense